<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExtra</class>
 <widget class="QDialog" name="DialogExtra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonHesapla">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Extra</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Room number :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_kola">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/cola.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fanta">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/fanta.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_kahve">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/kahve.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cay">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/cay.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_bos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>İçecekler:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Atıştırmalık</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gofret">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/gofret.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_kraker">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/kraker.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cips">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/cips.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>90</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dondurma">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/Dondurma.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Yemekler :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Hamburger">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/hamburger.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mant">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/mant.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_kebab">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/kebab.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pizza">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Images/pizza.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Toplam Fiyat :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fiyat">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Okey</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
